packages/hint-no-vulnerable-javascript-libraries/src/snyk-snapshot.json

```diff
@@ -1 +1 @@
8-11-18T11:50:41.399407Z","moduleName":"d3.js","packageManager":"npm","packageName":"d3.js","patches":[],"publicationTime":"2017-08-02T13:08:42.301000Z","references":[{"title":"Malicious packages published on npm","url":"https://iamakulov.com/notes/npm-malicious-packages/"},{"title":"Typosquatting programming language package managers","url":"http://incolumitas.com/2016/06/08/typosquatting-package-managers/"}],"semver":{"vulnerable":["<= 1.0.2"]},"severity":"high","title":"Malicious Package"}],"datatables":[{"CVSSv3":"CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:L/A:N","alternativeIds":["SNYK-JS-DATATABLES-10439"],"creationTime":"2017-03-20T13:17:55.262000Z","credit":["Allan Jardine"],"cvssScore":6.5,"disclosureTime":"2015-11-05T22:00:00Z","exploit":"Not Defined","fixedIn":["1.10.10"],"functions":[],"functions_new":[],"id":"npm:datatables:20151106","identifiers":{"ALTERNATIVE":["SNYK-JS-DATATABLES-10439"],"CVE":[],"CWE":["CWE-79"]},"language":"js","modificationTime":"2019-05-30T14:48:30.859743Z","moduleName":"datatables","packageManager":"npm","packageName":"datatables","patches":[],"publicationTime":"2017-05-08T12:34:45.631000Z","references":[{"title":"GitHub Commit","url":"https://github.com/DataTables/DataTables/commit/6f67df2d21f9858ec40a6e9565c3a653cdb691a6"}],"semver":{"vulnerable":["<1.10.10 >=1.10.1"]},"severity":"medium","title":"Cross-site Scripting (XSS)"},{"CVSSv3":"CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:L/A:N","alternativeIds":["SNYK-JS-DATATABLES-10042"],"creationTime":"2015-09-18T09:29:10Z","credit":["Onur Yilmaz"],"cvssScore":6.5,"disclosureTime":"2015-09-18T09:29:10Z","exploit":"Not Defined","fixedIn":["1.10.10"],"functions":[],"functions_new":[],"id":"npm:datatables:20150918","identifiers":{"ALTERNATIVE":["SNYK-JS-DATATABLES-10042"],"CVE":["CVE-2015-6584"],"CWE":["CWE-80"],"NSP":[5]},"language":"js","modificationTime":"2019-05-30T14:12:51.155705Z","moduleName":"datatables","packageManager":"npm","packageName":"datatables","patches":[],"publicationTime":"2015-09-18T09:29:10Z","references":[{"title":"GITHUB.COM","url":"https://github.com/DataTables/DataTables/issues/602"},{"title":"GITHUB.COM","url":"https://github.com/DataTables/DataTablesSrc/commit/ccf86dc5982bd8e16d"},{"title":"WWW.NETSPARKER.COM","url":"https://www.netsparker.com/cve-2015-6384-xss-vulnerability-identified-in-datatables/"}],"semver":{"vulnerable":["<1.10.10"]},"severity":"medium","title":"Cross-site Scripting (XSS)"}],"deap":[{"CVSSv3":"CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:L/A:L","alternativeIds":["SNYK-JS-DEAP-12118"],"creationTime":"2018-04-15T20:11:17.541000Z","credit":["Olivier Arteau (HoLyVieR)"],"cvssScore":7.3,"disclosureTime":"2018-04-15T20:11:17Z","exploit":"Not Defined","fixedIn":["1.0.1"],"functions":[],"functions_new":[],"id":"npm:deap:20180415","identifiers":{"ALTERNATIVE":["SNYK-JS-DEAP-12118"],"CVE":["CVE-2018-3749"],"CWE":["CWE-400"],"NSP":[611]},"language":"js","modificationTime":"2019-04-14T11:23:05.788690Z","moduleName":"deap","packageManager":"n
```

Comment:
Did you mean to update the snapshot as part of this PR?

Reply:
It gets updated on each build 😓
Need to find a better way that also works with the dependency linter.